android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/FormUtils.java

```diff
@@ -896,6 +899,32 @@ public JSONArray concatArray(JSONArray... arrs)
         return result;
     }
 
+    /**
+     * Get Bitmap given a location in the assets folder and the name of the file
+     *
+     * @param context
+     * @param folder The folder within the Assets folder where the image resides
+     * @param imageFile Path and the name of the file
+     * @return Bitmap
+     */
+    public static Bitmap getBitmap(Context context, String folder, String imageFile) {
+        Bitmap bitmap = null;
+        String filePath;
+        try {
+            if (folder == null || folder.isEmpty()){
+                filePath = DEFAULT_FORM_IMAGES_FOLDER + imageFile;
+            }
+            else {
```

Comment:
Could the else block move to line 916? It follows more recent coding style guidelines.

Reply:
Also, should you add a check for whether imageFile is empty?

Reply:
This is checked in ImageViewFactory line 68, but yes it is important to handle checking for whether the **imageFile** is empty since it is Util method that could be used elsewhere

Reply:
Cool